rpcs3/Crypto/aes.cpp

```diff
@@ -31,6 +31,20 @@
 
 #include "aes.h"
 
+#include <bitset>
+#include <cstdio>
+
+// Returns whether AES-NI is supported
+static bool AESNI() {
+	int cpuInfo[4];
+#ifdef _WIN32

```

Comment:
What do you mean, specifically? I ifdeffed the AESNI function so it would work on MSVC, gcc and clang.